tests/css_case/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -349,7 +349,7 @@ $KeepScssVar: val;
   }
 
   @KeepDetachedRuleset();
-  @KeepDetachedRuleset();
+  @keepdetachedruleset (/* comment */);
```

Comment:
This shouldn't have been lowercased, should it?

Reply:
@lydell good catch, thanks!